lib/twap/events/life_start.js

```diff
@@ -29,6 +29,10 @@ const onLifeStart = async (instance = {}) => {
     }
   }
 
+  // TODO: use await here once awaitable feature is added
+  // so first tick can operate on data already
+  subscribeDataChannels(state)
```

Comment:
lgtm. 
It seems that in this place, despite the issue with the lack of await, it behaves like the implementation in the master branch